### DISCUSSION TITLE
lfs 大文件获取失败

### DISCUSSION THREAD

**Discussion by doubleYang1020** [Tencent/tgfx#227] [General] [answered]
如下报错
batch response: This repository exceeded its LFS budget. The account responsible for the budget should increase it to restore access.
Failed to fetch some objects from 'https://github.com/libpag/tgfx.git/info/lfs'
node:child_process:964
    throw err;
    ^

Error: Command failed: depsync
batch response: This repository exceeded its LFS budget. The account responsible for the budget should increase it to restore access.
Failed to fetch some objects from 'https://github.com/libpag/tgfx.git/info/lfs'
帮忙看看这个怎么解决啊

**Comment by domchen** [comment]
你好像检出的项目地址错了，tgfx在Tencent这个组织下面。

**Reply by domchen** [reply]
我测试了一下Tencent组织下的也检出不了，我们这边联系一下公司看怎么解决。

**Reply by doubleYang1020** [reply]
谢谢大佬，等待您的回复。

**Reply by domchen** [reply, marked as answer]
已经恢复了。
